editor/io_plugins/editor_atlas.cpp

```diff
@@ -45,23 +46,23 @@ struct _EditorAtlasWorkRectResult {
 	int max_h;
 };
 
-void EditorAtlas::fit(const Vector<Size2i> &p_rects, Vector<Point2i> &r_result, Size2i &r_size) {
+static int nearestPow2(int d) {
```

Comment:
You could use one of those instead: https://github.com/godotengine/godot/blob/1a92906b681735a5a2acfd98e3d6a4c75a7f6115/core/typedefs.h#L167-L194